pkg/ebpf/events_processor.go

```diff
@@ -317,17 +317,18 @@ func (t *Tracee) processEvent(event *trace.Event) error {
 		}
 
 	case CgroupRmdirEventID:
+		cgroupId, err := getEventArgUint64Val(event, "cgroup_id")
+		if err != nil {
+			return fmt.Errorf("error parsing cgroup_rmdir args: %v", err)
+		}
+
 		if t.config.Capture.NetPerContainer {
-			pcapContext, _, err := t.getPcapContext(uint32(event.HostThreadID))
```

Comment:
the problem was that the `cgroup_rmdir` event doesn't happen in the context of the container (`containerId == ""`).
this means that the process (`event.HostThreadID`) triggering this event also have `containerId == ""`. trying to get the pcap context of this `HostThreadID` would look for the `containerId` field, realize that it's empty, and assume that the pcap context is of the host (TRACEE_OUT/out/host/capture.pcap). therefore it will close the pcap of the host, instead of the pcap of the container. 

the fix: use the `cgroup_id` argument of `cgroup_rmdir` event, to get the `container id` which is being closed. get the pcap context using this `container id`, and close the correct pcap file.